.github/workflows/automaton.yml

```diff
@@ -90,7 +90,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        map: ["COGMAP", "COGMAP2", "CLARION", "DESTINY", "DONUT3", "OSHAN", "KONDARU", "ATLAS", "SPACE_PREFAB", "UNDERWATER_PREFAB", "FULL"]
+        map: ["COGMAP", "COGMAP2", "CLARION", "DESTINY", "DONUT3", "DONUT2", "OSHAN", "KONDARU", "ATLAS", "MANTA", "SPACE_PREFAB", "UNDERWATER_PREFAB", "FULL"]
```

Comment:
nice try

Reply:
my PR was gonna handle this all but nooooo >:( 

I'll fix the merge issue :)

Reply:
should be fixed in [5c86fc2](https://github.com/goonstation/goonstation/pull/10285/commits/5c86fc2d85672abed8a25c6fe20fba1e486d4add)